packages/spa-manager/package.json

```diff
@@ -18,6 +18,8 @@
   },
   "devDependencies": {
     "@patternfly/react-core": "3.120.8",
+    "@patternfly/react-icons": "^3.14.21",
+    "@patternfly/react-styles": "^3.6.6",
```

Comment:
@kunyan I added these deps in order to get spa-manager's tests to pass.  Let me know if this looks good to you, or if I missed something.

Reply:
Looks good